nixos/modules/services/networking/mosquitto.nix

```
@@ -233,15 +232,50 @@ in
         ExecStart = "${pkgs.mosquitto}/bin/mosquitto -c ${mosquittoConf}";
         ExecReload = "${pkgs.coreutils}/bin/kill -HUP $MAINPID";
 
-        ProtectSystem = "strict";
-        ProtectHome = true;
+        # Hardening
+        CapabilityBoundingSet = "";
+        DevicePolicy = "closed";
+        LockPersonality = true;
+        MemoryDenyWriteExecute = true;
+        NoNewPrivileges = true;
         PrivateDevices = true;
+        PrivateUsers = true;
         PrivateTmp = true;
```

I assume your intention was alphabetic ordering.
```suggestion
        PrivateTmp = true;
        PrivateUsers = true;
```

---

Somewhat, yeah.